UTCI stress categories

grjonathan opened the pull request:
Issue #12
This adds the UTCI stress category (a string) to the output of models.utci.
Alternatively, this could be split into a separate function so that the output of models.utci remains a float.

FedericoTartarini commented:
Thank you for working on this but we need to make sure we do not break the code for people who are already using the UTCI old code. We could either: 
* add a warning stating that the UTCI code has changed 
* pass an input like `return_stress_categories` which is False by default. 
* create an additional function in utilities which calculates the stress categories.

grjonathan commented:
Thank you for having a look. I agree this feature shouldn't break existing applications nor the examples/tutorials. A separate function in utilities might make sense, but there would have to be a toggle for units. If users are piping outputs directly from models.utci, they might forget to set the units a second time.

I added a `return_stress_category` input set to False by default and reverted the test. Please let me know what you think.
